packages/block-editor/src/components/link-control/style.scss

```diff
@@ -150,14 +161,18 @@ $block-editor-link-control-number-of-actions: 1;
 		white-space: nowrap;
 	}
 
+	.block-editor-link-control__search-item-title mark {
+		color: $dark-gray-900;
+	}
+
 	.block-editor-link-control__search-item-title {
 		display: block;
 		margin-bottom: 0.2em;
 		font-weight: 500;
 
 		mark {
 			font-weight: 700;
-			color: #000;
+			color: $black;
 			background-color: transparent;
 		}
```

Comment:
These are addressing the same selector. Do we want the `mark` to be `$black` or `$dark-gray-900`? Note, I think this was my fault originally 😬